src/main/scala/com/mwz/sonar/scala/qualityprofiles/RecommendedQualityProfile.scala

```diff
@@ -28,23 +28,22 @@ import org.sonar.api.server.profile.BuiltInQualityProfilesDefinition.NewBuiltInQ
 /**
  * Defines a quality profile recommended by sonar-scala (including Scalastyle and Scapegoat).
  */
-final class RecommendedQualityProfile extends BuiltInQualityProfilesDefinition {
+final class RecommendedQualityProfile(
+  scalastyleRulesRepository: ScalastyleRulesRepository
```

Comment:
Why do you need the `ScalastyleRulesRepository` ?
I don't see it anywhere in the code. Also, if you really need it, remember we need to inject it using DI.